src/README.md

```diff
@@ -0,0 +1,48 @@
+# LibEzINI
```

Comment:
This file is almost but not quite a copy of the README.md in the root. I don't think it is good to keep both, in the interest of maintainability – they have already diverged.

Reply:
I am repeating sum but not all comments on that file here, so please make sure to take all comments into account, whether you keep one or both files.

Reply:
sure, and good point!

Reply:
ok removed the file, so I guess this is resolved